doc/chilled.rst

```diff
@@ -0,0 +1,67 @@
+.. currentmodule:: message_ix_buildings.chilled
+
+CHILLED (:mod:`.chilled`)
+*************************
+
+:mod:`.chilled` (Cooling and Heating gLobaL Energy Demand) is a model that assesses the impacts of climate change (temperature change)
+
+Methods
+=======
+
+
+Data
+====
+
+Input data
+----------
+
+
+Parameters
+----------
+
+In this module, the "version" refers to the version of the model, not the version of the data. For example, the "ALPS2023" version refers to the set of scenario inputs/parameters for the "ALPS2023" run.
+When specifying a version, CHILLED will look in the `/data/chilled/version` directory that the version name exists and that the following files exist as well:
```

Comment:
```suggestion
When specifying a version, CHILLED will look in the `/data/chilled/version` directory that the version name exists and that the following files exist as well:

```

This does not render as a proper list (which I presume is the intention). I think a blank line before the list should fix this.

To check how it renders, use a venv where you have installed all dependencies you need to render e.g. the message docs. You should then be able to run `sphinx-build -T -E -b html -d _build/doctrees-readthedocs -D language=en . _build/html` inside the `doc` directory and can navigate to the pages when hosting them yourself via `python -m http.server` :)